package.json

```diff
@@ -44,10 +44,13 @@
   },
   "homepage": "https://github.com/ldthomas/apg-js#readme",
   "devDependencies": {
+    "browserify": "17.0.0",
+    "minify": "10.2.0",
```

Comment:
these should probably be inserted in alphabetical order

Reply:
pushed an update to sort correctly but for some reason the PR is not reflecting this